lib/components/Stack/Stack.test.tsx

```diff
@@ -0,0 +1,62 @@
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { BraidProvider, Stack, Text } from '..';
+import { wireframe } from '../../themes';
+
+afterEach(cleanup);
+
+describe('Stack', () => {
+  it('should not render a list by default', () => {
+    const { queryAllByRole } = render(
+      <BraidProvider theme={wireframe}>
+        <Stack space="small">
+          <Text>1</Text>
+          <Text>2</Text>
+          <Text>3</Text>
+        </Stack>
+      </BraidProvider>,
+    );
+
+    expect(queryAllByRole('list').length).toBe(0);
```

Comment:
Does `queryAllByRole` rely on there being an aria attribute on the element? Or will it return the element if it's simply a list?

Reply:
It supports both 😄